src/Arcus.BackgroundJobs.Tests.Integration/Arcus.BackgroundJobs.Tests.Integration.csproj

```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Arcus.EventGrid.Publishing" Version="3.0.0" />
-    <PackageReference Include="Arcus.EventGrid.Testing" Version="3.0.0" />
-    <PackageReference Include="Arcus.Security.Providers.AzureKeyVault" Version="1.3.0" />
+    <PackageReference Include="Arcus.EventGrid.Publishing" Version="3.2.0-preview-1" />
```

Comment:
Can we ship v3.2.0?

Reply:
Yeah, was waiting on https://github.com/arcus-azure/arcus.eventgrid/pull/175 before shipping.